plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -629,6 +629,15 @@ public WriteMapping toWriteMapping(ConnectorSession session, Type type)
             return WriteMapping.objectMapping(dataType, longTimestampWriteFunction(timestampType, precision));
         }
 
+        if (type instanceof TimestampWithTimeZoneType timestampWithTimeZoneType) {
```

Comment:
Please update `BaseSqlServerTypeMapping`. I don't remember why I didn't support writes in my previous PR though.

Reply:
Added